packages/flutter_tools/gradle/aar_init_script.gradle

```diff
@@ -0,0 +1,117 @@
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.maven.MavenDeployer
+import org.gradle.api.plugins.MavenPlugin
+import org.gradle.api.tasks.Upload
+
+void configureProject(Project project, File outputDir) {
+    if (!project.hasProperty("android")) {
+        throw new GradleException("Android property not found.")
+    }
+    if (!project.android.hasProperty("libraryVariants")) {
+        throw new GradleException("Can't generate AAR on a non Android library project.");
+    }
+
+    project.apply plugin: "maven"
+
+    project.android.libraryVariants.all { variant ->
+        addAarTask(project, variant)
+    }
+    // Snapshot versions include the timestamp in the artifact name.
+    project.version = project.version.replace("-SNAPSHOT", "")
```

Comment:
Are we sure we want to remove this?  I believe that's set by default for plugins, but what if a plugin author explicitly wanted this?

Reply:
Talked offline and it sounds like this is the right way to go - but maybe you can expand the comment here explaining why we're convinced this works beyond just that our templates happen to include it today in our templates.

Reply:
Solved offline :)